Updating Shared Symbology

CHafen opened the pull request:
Changed the BL symbology "DEM" to "dem" for topo and hydro 
removed DEM.lyr files for topo and hydro. Removed wsDEM.lyr from hydro as it is already in shared. Moved detrendDEM.qml from topo to shared.
Updated cu_T2 symbology for topo

joewheaton commented:
I want to make sure this doesn't mess up our many other project types that use DEM.lyr... if it does we will rename these reach scale ones. It might not...

CHafen commented:
@joewheaton the shared the symbology file is "dem.lyr" and "dem.qml"  I didn't change those. In the topo and hydro BL it was "DEM" so I changed the business logic for them to "dem" so it would use the shared symbology. And they had their own "DEM" symbology files so I deleted the "DEM.lyr" and "DEM.qml" files from **hydro and topo symbology** so it would just use the symbology from the shared layer files. So, for those I didn't actually change anything in the shared symbology just in the hydro and topo. I added detrendDEM to the shared Q symbology and wsDEM to the shared Arc and Q symbology.

joewheaton commented:
Awesome. Thank you for clarification.
